### PR TITLE
fix: CLIN-2300 disable export tsv button when total is 0

### PR DESCRIPTION
**Pull request by meek0**
# FIX

- closes CLIN-2300

## Description

[JIRA LINK](https://ferlab-crsj.atlassian.net/browse/CLIN-2300)

Acceptance Criterias

## Validation

- [ ] Storybook add or modified
- [ ] version Update in package.json and Release.md
- [ ] Code Approved
- [ ] QA Done
- [ ] Design/UI Approved from design

## QA

Steps to validate
Url (storybook, ...)
...

## Mention

@kstonge @luclemo

### CODE REVIEW

File: packages/ui/src/components/ProTable/index.tsx [hunk 1]
```diff
@@ -194,6 +194,7 @@ const ProTable = <RecordType extends object & { key: string } = any>({
             customExtra = customExtra.concat(
                 <Tooltip title={dictionary.tooltips?.tableExport || 'Export as TSV'}>
                     <Button
+                        disabled={(headerConfig.itemCount?.total || 0) === 0}
```
Comment (ethienneroy) [inline packages/ui/src/components/ProTable/index.tsx:197]: is it really this package responsibility to disable it ? or should it come from the app using the library ?

File: packages/ui/Release.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ### 8.2.1 2024-02-02
-- fix: SJIP-680 fix config corrumption when resizing the browser 
+- fix: SJIP-680 fix config corrumption when resizing the browser
```
Comment (ethienneroy) [inline packages/ui/Release.md:2]: missing release version and description

**Review by ethienneroy — CHANGES_REQUESTED** [review]

File: packages/ui/Release.md (hunk 2 above)
Comment (ethienneroy) [reply packages/ui/Release.md:2]: version already exists

File: packages/ui/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@ferlab/ui",
-    "version": "8.2.3",
+    "version": "8.2.4-rc1",
```
Comment (ethienneroy) [inline packages/ui/package.json:3]: version already exists

File: packages/ui/src/components/ProTable/index.tsx (hunk 1 above)
Comment (meek0) [reply packages/ui/src/components/ProTable/index.tsx:197]: unless disabled state makes sense for another reason other then "no data", I think the package can take care of it

Comment (meek0) [reply packages/ui/src/components/ProTable/index.tsx]: added headerConfig property

File: packages/ui/Release.md (hunk 2 above)
Comment (meek0) [reply packages/ui/Release.md:5]: added entry

File: packages/ui/package.json (hunk 3 above)
Comment (meek0) [reply packages/ui/package.json]: sync with version in package.json

File: packages/ui/Release.md [hunk 4]
```diff
@@ -1,3 +1,6 @@
+### 8.2.1 2024-02-02
```
[the remark this answers is not in the document]
Comment (meek0) [reply packages/ui/Release.md]: sync with version in package.json

**Review by ethienneroy — APPROVED** [review]

**Pull request merged**